styles/index.js

```diff
@@ -1,20 +1,4 @@
-var fs = require('fs')
-var path = require('path')
-var compile = require('micro-css')
-var result = ''
-var additional = ''
-
-fs.readdirSync(__dirname).forEach(function (file) {
-  if (/\.mcss$/i.test(file)) {
-    result += fs.readFileSync(path.resolve(__dirname, file), 'utf8') + '\n'
-  }
-
-  if (/\.css$/i.test(file)) {
-    additional += fs.readFileSync(path.resolve(__dirname, file), 'utf8') + '\n'
-  }
-})
-
-const flatpickr = require.resolve('flatpickr/dist/flatpickr.css')
-additional += fs.readFileSync(flatpickr) + '\n'
-
-module.exports = compile(result) + additional
+module.exports = {
+  light: require('./light'),
+  dark: require('./dark')
```

Comment:
I think it's worth getting this code merged asap. Since all the CSS files have changed location this branch is quickly going to become conflicty with other branches that need to change the CSS.

Reply:
merged into your ahhh dark branch right?

if yes then go for it